docs/source/zero-code-change-limitations.rst

```diff
@@ -1,21 +1,32 @@
-cuml.accel: Known Limitations
-=============================
+Known Limitations
```

Comment:
Can you do me a favor and revert anything that touches the general sections so we can limit this PR just to the KMeans information? I'm 100% down with discussing tweaks to the more general info, but I'd prefer to do it once on the overall PR so we're not re-addressing it with each PR for individual algorithm.

Reply:
The main motivation came from having to jiggle things around to make the docs build. Can revert them here but I think we should make a first (or zeroth?) PR that gets the docs to build and then add content.

Reply:
Good enough motivation for me! The repeated `cuml.accel` may be something we need to find a workaround for. I'd like to have that sprinkled among our titles as liberally as possible for search-friendliness. Let's merge this, and I'll look at ways to make that happen.